### PR TITLE
Absences page

### PR DESCRIPTION
**Pull request by saherhanif**

### CODE REVIEW

**Review by LeenMass — DISMISSED** [review]

**Comment by LeenMass** [comment]
you must to run npm run prettier:fix to pass the ci

**Review by MuhammadAw90 — APPROVED** [review]

**Comment by MuhammadAw90** [comment]
Nice job !
Just run the CI

**Review by Karyum — APPROVED** [review]

**Comment by Karyum** [comment]
@tawfiknasser you forgot about this PR

File: src/pages/Absence/index.jsx [hunk 1]
```diff
@@ -1,5 +1,49 @@
...
+import PageContainer from '../../components/PageContainer'
+import SearchBar from '../../components/SearchBar'
 
 export default function Absence() {
-  return <div>Absence</div>
+  const [list, setList] = React.useState([])
+
+  const getAbsence = async () => {
+    try {
+      const data = await fetch('http://localhost:4000/absence', {
```
Comment (tawfiknasser) [inline src/pages/Absence/index.jsx:11]: Use here the api from the config as you did in the other file...

This will make our life easier in the host task

**Comment by tawfiknasser** [comment]
@saherhanif fix the comment and the ci and this should be merged

**Pull request closed without merging**